appyters/ChEA3_Appyter/README.md

```diff
@@ -2,10 +2,10 @@
 
 This is an example appyter in the catalog. Please reference it when submitting your own appyter.
 
-- `{name}/{name}.ipynb`: The appyter iteself
-- `{name}/README.md`: Long description of appyter
+- `{name}/{name}.ipynb`: ChEA3 Appyter
+- `{name}/README.md`: The ChEA3 Appyter (ChIP-X Enrichment Analysis 3) predicts transcription factors (TFs) associated with user-input sets of genes. Discrete query gene sets are compared to ChEA3 libraries of TF target gene sets assembled from multiple orthogonal 'omics' datasets. The Fisher's Exact Test, with a background size of 20,000, is used to compare the input gene set to the TF target gene sets in order to determine which TFs may be most closely associated with the input gene set. Users can submit a set of human or mouse gene symbols for transcription factor enrichment analysis.
```

Comment:
This is what this looks like
https://github.com/MaayanLab/appyter-catalog/blob/d5807b2de1962a82cc53df2f276106df21ecdc4b/appyters/ChEA3_Appyter/README.md

Please see, for example: https://github.com/MaayanLab/appyter-catalog/blob/master/appyters/Drugmonizome_ML/README.md

Listing requirements here is not needed, that information is in that file. the example README is just there to describe the example, not to be built off of -- just replaced.